### PR TITLE
BOAC-188, orderBy on cohort view; db mod scripts; tweak to load_cohorts_from_asc.py

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/BOAC-188

* Script to load names from CSV updated
* Pre- and post-migration scripts (SQL)
* Front-end changes per orderBy first and last name

### CODE REVIEW

File: boac/externals/asc_cohorts.py [hunk 1]
```diff
@@ -53,9 +53,13 @@ def load_cohort_from_csv(app, csv_file='tmp/FilteredAscStudents.csv'):
...
                 continue
             sis_sport_code = SPORT_TRANSLATIONS[asc_sport_code_core]
+            name_split = r['cName'].split(',') if 'cName' in r else ''
+            full_name = [name.strip() for name in reversed(name_split)]
+            # Construct an athlete
             record = TeamMember(
                 member_csid=r['SID'],
-                member_name=r['cName'],
+                first_name=full_name[0].strip() if len(full_name) else '',
+                last_name=full_name[1].strip() if len(full_name) > 1 else '',
```
Comment (johncrossman) [inline boac/externals/asc_cohorts.py]: The above was tested on CSV export of the golden ASC spreadsheet. Names landed correctly.

File: boac/models/team_member.py [hunk 2]
```diff
@@ -177,10 +181,15 @@ def get_team_groups(cls, team_group_codes):
         athletes = cls.query.distinct(cls.asc_sport_code).filter(query_filter).all()
         return [athlete.team_group_summary() for athlete in athletes]
 
+    @classmethod
+    def translate_param(cls, order_by):
```
Comment (paulkerschen) [inline boac/models/team_member.py]: @johncrossman Should this have a more descriptive name than `translate_param` - `get_ordering` or something of the sort?

**Comment by paulkerschen** [comment]
Looks good to me; one minor comment above.

File: boac/models/team_member.py (hunk 2 above)
Comment (johncrossman) [reply boac/models/team_member.py]: I had a long, descriptive name then changed it to this short and vague. I'll find a middle ground. Thx, @paulkerschen

**Pull request merged**